### DISCUSSION TITLE
HTML live server support?

### DISCUSSION THREAD

**Discussion by prophesierc** [coder/code-server#5409] [Q&A]
Does anyone know how to get a live web server like the live server or live preview extensions to work for HTML? I'm running it from Nginx with Let's Encrypt.

Any tutorials or does anyone know how to explain it like I'm 5?

**Comment by jsjoeio** [comment]
This has been asked a couple times. I'd dig into those Discussions [here](https://github.com/coder/code-server/discussions?discussions_q=live+preview) and see if you find a solution
